AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusElementHighlight.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.accessibilityinsightsforandroidservice;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.View;
+import android.view.accessibility.AccessibilityNodeInfo;
+import java.util.HashMap;
+
+public class FocusElementHighlight {
+  private static final String TAG = "FocusElementHighlight";
+  private AccessibilityNodeInfo eventSource;
+  private int yOffset;
+  private int tabStopCount;
+  private int radius;
+  private int xCoordinate;
+  private int yCoordinate;
+  private HashMap<String, Paint> paints;
+  private Rect rect;
+  private View view;
+
+  public FocusElementHighlight(
+      AccessibilityNodeInfo eventSource,
+      HashMap<String, Paint> currentPaints,
+      int radius,
+      int tabStopCount,
+      View view) {
+    this.view = view;
+    this.eventSource = eventSource;
+    this.yOffset = setYOffset();
```

Comment:
Would it make sense to call `updateWithNewCoordinates` here, instead of having separate calls to `setYOffset` and `setCoordinates` ? That gives us a single place where we handle any position-related stuff

Reply:
This was a good suggestion, I've updated the code to do this.